src/backer/cardaning.py

```diff
@@ -49,24 +49,22 @@ class Cardano:
 
     def __init__(self, hab, ks=None):
         self.pending_kel = bytearray()
-        self.keldbConfirming = subing.Suber(db=hab.db, subkey="kel_queued")
+        self.keldbConfirming = subing.Suber(db=hab.db, subkey="kel_confirming")
         self.context = pycardano.OgmiosV6ChainContext()
         self.client = ogmios.Client()
         self.tipHeight = 0
 
-        # retrieve backer private key and derive cardano address
-        backer_private_key = ks.pris.get(hab.kever.prefixer.qb64).raw
-        self.payment_signing_key = pycardano.PaymentSigningKey(backer_private_key, "PaymentSigningKeyShelley_ed25519", "PaymentSigningKeyShelley_ed25519")
-        payment_verification_key = pycardano.PaymentVerificationKey.from_signing_key(self.payment_signing_key)
-        self.spending_addr = pycardano.Address(payment_part=payment_verification_key.hash(),staking_part=None, network=NETWORK)
-        print("Cardano Backer Address:", self.spending_addr.encode())
+        self.payment_signing_key = pycardano.PaymentSigningKey.from_cbor(os.environ.get('FUNDING_ADDRESS_CBORHEX'))
```

Comment:
Lets change the env to `WALLET_ADDRESS_CBORHEX` throughout the repo. And update any variable names here. It's no longer the funding address, it's now the spending address